pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -103,13 +104,19 @@ func (grm *nestedPendingOperations) Run(
 		// Operation already exists
 		if previousOp.operationPending {
 			// Operation is pending
-			operationName := getOperationName(volumeName, podName)
-			return NewAlreadyExistsError(operationName)
+			uniquePodVolumeName := getOperationName(volumeName, podName)
```

Comment:
I renamed these because what we are generating here is not truly operation names but unique operation keys that we have been using to prevent concurrent operation.  I guess I should rename all onccurences of "operationName" in existing code to "operationKey" or something to prevent confusion with "real" operationName.

Reply:
Could you modify the comments in Run https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go#L48 to reflect the changes?

Reply:
It might be a good idea to rename `getOperationName` as well.

Reply:
@bertinatto @jingxu97 addressed both your comments.